### ISSUE TITLE
Req: ability to open current image in browser

### ISSUE THREAD

**Issue by lionkmp** [JOSM/Mapillary#194]
There is an icon to copy the URL and then I can paste that in the browser.
But it would be faster to do with a hotkey or with an icon. If possible.
Something like Ctrl+Shift+H for history or Ctrl+Shift+I for info.
Thanks.

**Comment by tsmock** [comment]
This already exists in the latest version of JOSM and the Mapillary plugin.
![Screen Shot 2022-04-29 at 5 29 59 AM](https://user-images.githubusercontent.com/45215054/165936525-d93e97fc-d6e6-458d-a0c0-d8ac6780388d.png)

**Comment by tsmock** [comment]
But is there any particular reason why you want to open the image in the browser? That seems like a more relevant problem for me to fix.

**Comment by lionkmp** [comment]
Sorry for reporting an invalid request. Actually I saw that icon, but since the tooltip is "**open image in external viewer**" I expected it would open the image in my image viewer, that I didn't want. So I didn't click it. :-)  (You know, that happens with geotagged images that are loaded as JPG from the PC.)

So I find that tooltip misleading.

Reason is to look around! Mapillary.com has better navigation than any similar systems. So to choose other views of the same object, walk around, etc, it's easier on the website than in JOSM. Thank you for asking.

**Comment by lionkmp** [comment]
If you could overlay it with the same navigation arrows as Mapillary.com has. Turn left, turn right, and so on. If that's possible. Then I would have little to no reason to open Mapillary.com.

And if possible, please change that tooltip to "Open image on Mapillary.com".

**Comment by tsmock** [comment]
> If you could overlay it with the same navigation arrows as Mapillary.com has. Turn left, turn right, and so on. If that's possible. Then I would have little to no reason to open Mapillary.com.
I'd like to do this, but I don't know how (exactly) the Mapillary website is getting those navigation arrows. I should probably spend some time looking at some of the API returns.

> And if possible, please change that tooltip to "Open image on Mapillary.com".
No can do, unfortunately. That string is set in JOSM once. I could probably make some changes to JOSM core to enable this use case, but it would take months (I try to maintain compatibility with the past three months of JOSM releases, and I prefer to keep compatibility for longer than that).

**Comment by lionkmp** [comment]
Yes, I expected these are not easy things, otherwise they would be in the plugin already. Thank you! Keep up the good work. The plugin is a big help in mapping.
